src/interactions/applicationCommand.ts

```diff
@@ -256,9 +256,12 @@ export class SlashBuilder {
 export function transformApplicationCommandOption(
   _data: ApplicationCommandOption
 ): ApplicationCommandOptionPayload {
-  const data = { ...(_data as any) }
+  const data = { ...(_data as unknown as Record<string, unknown>) }
```

Comment:
out of curiosity. is it not possible to change this to the following
```suggestion
  const data: Record<string, unknown> = { ..._data }
```

Reply:
Yep.. I'll do it.